tests/IO/IOCSVFileIteratorTest.php

```diff
@@ -1,9 +1,9 @@
 <?php
-
-require_once dirname(__FILE__) . '/../../IO/IOCSVFileIterator.php';
+require_once dirname(__FILE__) . "/../../vendor/autoload.php";
 require_once dirname(__FILE__) . '/../dependencies/vendor/autoload.php';
```

Comment:
@kathyoungaligent Should we need line 3 since there is line 2 added in now

Reply:
@ramya-aligent - there are two different vendor folders and I think it needs the autoload from both of them?